.github/workflows/deploy_ghpackages_docker.yml

```
@@ -0,0 +1,23 @@
+name: deploy github packages (docker)
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Build the Docker image
+        run: docker build . -t docker.pkg.github.com/planetarium/ninechronicles.standalone/ninechronicles-headless:git-${{ github.sha }} --build-arg COMMIT=git-${{ github.sha }} -t docker.pkg.github.com/planetarium/ninechronicles.standalone/ninechronicles-headless:latest --build-arg COMMIT=git-${{ github.sha }}
```

I wonder that it needs the same build arguments twice?
```suggestion
        run: docker build . -t docker.pkg.github.com/planetarium/ninechronicles.standalone/ninechronicles-headless:git-${{ github.sha }} -t docker.pkg.github.com/planetarium/ninechronicles.standalone/ninechronicles-headless:latest --build-arg COMMIT=git-${{ github.sha }}
```

---

oops, I completely forgot about this review. Thanks! 😅